docs/transformation_grids.rst

```diff
@@ -0,0 +1,92 @@
+.. _transformation_grids:
+
+Transformation Grids
+=====================
+
+Transformation grids are necessary when you are performing datum transformations.
+
+More information about the data available is located under the PROJ
+`resource files <https://proj.org/resource_files.html#transformation-grids>`__
+documentation.
+
+`pyproj` API for managing the :ref:`data_directory`
+
+.. warning:: pyproj 2 includes datumgrid 1.8 in the wheels. pyproj 3 will not include any datum grids.
+
+
+Downloading data
+----------------
+
+PROJ 7+
+^^^^^^^^
+
+PROJ 7.0 has introduced, per
+`PROJ RFC 4: Remote access to grids and GeoTIFF grids <https://proj.org/community/rfc/rfc-4.html#rfc4>`__,
+the capability to work with grid files that are not installed on the local machine where PROJ is executed.
+
+Available methods for download include:
+
+- `Mirroing the data <https://proj.org/usage/network.html#mirroring>`__:
+
+  .. code-block:: bash
+
+    aws s3 sync s3://cdn.proj.org $(python -c "import pyproj; print(pyproj.datadir.get_data_dir())")
+
+  .. code-block:: bash
+
+    wget --mirror https://cdn.proj.org/ -P $(python -c "import pyproj; print(pyproj.datadir.get_data_dir())")
+
+- The `projsync <https://proj.org/apps/projsync.html>`__ command line program.
+
+- Enabling `PROJ network <https://proj.org/usage/network.html>`__ capabilities.
+
+- Use `conda <https://conda.io/en/latest/>`__ with the `conda-forge <https://conda-forge.org/>`__ channel:
+
+  .. code-block:: bash
+
+     conda install -c conda-forge proj-data
+
+
+PROJ <= 6
+^^^^^^^^^^
+
+Available methods for download include:
+
+- Download stable from https://download.osgeo.org/proj or latest from https://github.com/OSGeo/proj-datumgrid
+
+- Use `conda <https://conda.io/en/latest/>`__ with the `conda-forge <https://conda-forge.org/>`__ channel:
+
+  .. code-block:: bash
+
+     conda install -c conda-forge proj-datumgrid-europe proj-datumgrid-north-america proj-datumgrid-oceania proj-datumgrid-world
+
+
+What grids to download?
+-----------------------
+
+- Only using the :obj:`pyproj.crs.CRS` or :obj:`pyproj.Geod` classes? Then no grids are needed.
+
+- Have a machine that can hold and extra 500 MB - 1 GB of data? Then downloading all grids shouldn't be an issue.
+
+- Have a machine with limited space, a great network connection, and PROJ 7+? Look into `PROJ network <https://proj.org/usage/network.html>`__ capabilities.
+
```

Comment:
With PROJ 7, it will also download the subset of grids you need (and cache those) automatically?

Reply:
By default it isn't enabled. You need to enable it. However, if you installed it via conda-forge it is enabled by default.